doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -773,3 +773,8 @@
     ```
 
     Have a look at the logger design document for more details on how to setup the testing logger.
+
+40. Changed the include path and namespace of several classes:
```

Comment:
Is your plan that we add all other moved classes to this section?

Reply:
@FerdinandSpitzschnueffler Yes or if we manage to do it completely for `v3.0` (meaning closing #1593), then we could also go for a generic example `s/namespace_foo::barClass/iox::barClass`